tests/test_utils/mod.rs

```diff
@@ -26,9 +26,19 @@ pub use sqlparser::test_utils::*;
 #[macro_export]
 macro_rules! nest {
     ($base:expr $(, $join:expr)*) => {
-        TableFactor::NestedJoin(Box::new(TableWithJoins {
+        TableFactor::NestedJoin { table_with_joins: Box::new(TableWithJoins {
             relation: $base,
             joins: vec![$(join($join)),*]
-        }))
+        }), alias: None}
+    };
+}
+
+#[macro_export]
+macro_rules! nest_with_alias {
```

Comment:
Since this macro is only invoked in a single place I can see, what do you think about inlining it instead?

Reply:
done